genesis/process/genesisBlockCreator.go

```diff
@@ -417,6 +418,7 @@ func (gbc *genesisBlockCreator) computeDNSAddresses() error {
 		DataPool:           gbc.arg.Data.Datapool(),
 		CompiledSCPool:     gbc.arg.Data.Datapool().SmartContracts(),
 		NilCompiledSCStore: true,
+		EpochNotifier:      forking.NewGenericEpochNotifier(),
```

Comment:
\+ `SaveAccountsIfErrorDisableEpoch: unreachableEpoch`

Reply:
done, although not needed :D